site/sfguides/src/data_collaboration_native_app/data_collaboration_native_app.md

```diff
@@ -0,0 +1,630 @@
+
+Select the CREDIT_CARD_PREDICTION_PACKAGE, and click save.
+
+Select "Access Control" to open up the modal. Under "Who can access this data product?" select "Selected accounts and roles". In the Accounts dropdown, select SNOWBANK.
+
+For the Allow Discovery section, select "Not discoverable by users without access".
+
+Click Save
+
+![Diagram](assets/package_listing_ui.png)
+
+Add a support email to the listing.
+
+Finally, click Publish.
+
+
+<!-- ------------------------ -->
+## Consume Model via Native App (Snowbank)
+Duration: 10
+
+In this next step, we will log in to our consumer account (Snowbank). You should have saved the details such as the account URL and username from step 3. Log in to the Snowbank Account, and follow the steps to reset your password.
+
+Next, lets ingest the data into this completely separate account by running the following commands in a SQL worksheet.
+
+### Setup and Load Data
+
+```SQL
+USE ROLE ACCOUNTADMIN;
+
+CREATE OR REPLACE WAREHOUSE QUERY_WH WITH 
+  WAREHOUSE_SIZE = 'X-SMALL' 
+  WAREHOUSE_TYPE = 'STANDARD' 
+  AUTO_SUSPEND = 300 
+  AUTO_RESUME = TRUE 
+  MIN_CLUSTER_COUNT = 1 
+  MAX_CLUSTER_COUNT = 1;
+
+CREATE OR REPLACE DATABASE NATIVE_APP_CONSUMER;
+CREATE OR REPLACE SCHEMA NATIVE_APP_CONSUMER;
+USE SCHEMA NATIVE_APP_CONSUMER.NATIVE_APP_CONSUMER;
+
+CREATE OR REPLACE FILE FORMAT parquet_format
+TYPE = PARQUET;
+
+CREATE OR REPLACE STAGE quickstart_cc_default_unscored_data
+    URL = 's3://sfquickstarts/two_way_data_share/unscored/'
+    FILE_FORMAT = parquet_format;
+
+CREATE OR REPLACE TABLE cc_default_unscored_data 
+  USING TEMPLATE (
+    SELECT ARRAY_AGG(OBJECT_CONSTRUCT(*))
+      FROM TABLE(
+        INFER_SCHEMA(
+          LOCATION=>'@quickstart_cc_default_unscored_data',
+          FILE_FORMAT=>'parquet_format'
+        )
+      ));
+
+ALTER WAREHOUSE query_wh SET warehouse_size=MEDIUM;
+
+COPY INTO cc_default_unscored_data 
+  FROM @quickstart_cc_default_unscored_data 
+  FILE_FORMAT = (FORMAT_NAME= 'parquet_format') 
+  MATCH_BY_COLUMN_NAME=CASE_INSENSITIVE;
+
+ALTER WAREHOUSE query_wh SET warehouse_size=XSMALL;
+```
+
+### Accept Shared Model
+
+On the left menu, navigate through to Data Products > Marketplace. Then select the Internal Marketplace tab. You should see your app ready to be installed.
+
+![Diagram](assets/listing_consumer_side_ui.png)
+
+Select Credit Card Default Model, and on the next screen, select Get. If you do not see the Get button, ensure that you are operating as the ACCOUNTADMIN role. Wait for it to install, and then click "Configure".
+
+Select the Privileges tab, and then click "Add" to let our App refer to data in the consumer account
+
+![Diagram](assets/consumer_app_config_ui.png)
+
+Click "Select Data" and navigate to the CC_DEFAULT_UNSCORED_DATA in the NATIVE_APP_CONSUMER database and the NATIVE_APP_CONSUMER schema. Click save.
+
+<!-- ------------------------ -->
+## Use Shared Model in Consumer Account (Snowbank)
```

Comment:
@sfc-gh-tbuchhorn Rename this to "Use Shared Model" and add the rest as sub-header underneath.